pisek/utils/terminal.py

```diff
@@ -25,6 +25,12 @@
 except OSError:
     terminal_width, terminal_height = 100, 24
 
+TARGET_LINE_WIDTH = min(terminal_width, 100)
+
+
+def right_aligned_text(left: str, right: str, offset: int = 0):
+    return pad(left, TARGET_LINE_WIDTH - len(right) + offset) + right
```

Comment:
Woudn't it better if this function ensured there was at least one space between `left` and `right`?
As far as I can tell, this needs to be done manually at all call sites.

Reply:
Great!
However, a space is still added manually at most callsites, which should be removed.